src/routers/alpha-router/gas-models/v3/v3-heuristic-gas-model.ts

```diff
@@ -53,11 +54,16 @@ export class V3HeuristicGasModelFactory extends IV3GasModelFactory {
     gasPriceWei: BigNumber,
     poolProvider: IV3PoolProvider,
     token: Token
+    // this is the quoteToken
   ): Promise<IGasModel<V3RouteWithValidQuote>> {
     // If our quote token is WETH, we don't need to convert our gas use to be in terms
     // of the quote token in order to produce a gas adjusted amount.
     // We do return a gas use in USD however, so we still convert to usd.
-    if (token.equals(WETH9[chainId]!)) {
+
+    // POLYGON TODO
```

Comment:
```suggestion
    // TODO: polygon
```
probably a more standard way of doing it

Reply:
also can remove?